lib/bundler/mirror.rb

```diff
@@ -0,0 +1,183 @@
+module Bundler
+  class Settings
+    class Mirrors
```

Comment:
I agree that it is not specific enough, but I honestly cannot think of any better name. Any suggestion?